profiles/utilities.py

```diff
@@ -1,5 +1,10 @@
 import random
 import string
+from datetime import datetime, timedelta
+
+
+def expires_at(expires_in: int) -> datetime:
```

Comment:
don't understand why the spec put in a `expires_in` which makes the expiration time inaccurate while they could have provided a precise `expires_at`